src/main/java/com/qaprosoft/carina/demo/gui/components/FooterMenu.java

```diff
@@ -34,6 +34,8 @@
     
     @FindBy(linkText = "News")
     private ExtendedWebElement newsLink;
+    
+    private String unUsedVar;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "unUsedVar" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube:9000/sonarqube/coding_rules#rule_key=squid%3AS1068)